src/pages/WizkidDetails.js

```diff
@@ -25,8 +25,12 @@ function WizkidDetails() {
         <WizkidElementContainer>
           <p className="wizkid-details-name">{found.name}</p>
           <p className="wizkid-details-position">{found.position}</p>
-          <p className="wizkid-details-phone-email">{found.phone}</p>
-          <p className="wizkid-details-phone-email">{found.email}</p>
+          {token && (
```

Comment:
Might be interesting to have a isLoggedIn value or something, is easier to read/understand if you implement it somewhere